Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Start from a Debian image with the latest version of Go installed
+# and a workspace (GOPATH) configured at /go.
+FROM marcbachmann/libvips:latest
+
+# Server port to listen
+ENV PORT 8080
+
+# Go version to use
+ENV GOLANG_VERSION 1.8.1
+
+# gcc for cgo
+RUN apt-get update && apt-get install -y \
+    gcc curl git libc6-dev make ca-certificates \
+    --no-install-recommends \
+  && rm -rf /var/lib/apt/lists/*
+
+ENV GOLANG_DOWNLOAD_URL https://golang.org/dl/go$GOLANG_VERSION.linux-amd64.tar.gz
```

Comment:
I'm not sure what you mean. From lines 9-29 are the only place we download and install go